libs/domain/picking/src/services/picking-http-default.service.ts

```diff
@@ -54,10 +48,22 @@ export class PickingHttpDefaultService implements PickingHttpService {
     options?: RequestOptions<T>
   ): Observable<T> {
     url = this.createFullUrl(url);
-    return this.httpService.patch(url, body, options);
+    return this.httpService.patch(url, body, this.expandContentType(options));
   }
 
   protected createFullUrl(url: string): string {
     return `${this.baseUrl}${url}`;
   }
+
+  protected expandContentType<T = unknown>(
+    options?: RequestOptions<T>
+  ): RequestOptions<T> {
+    return {
+      ...options,
+      headers: {
+        ...options?.headers,
+        'Content-Type': 'application/vnd.api+json',
+      },
```

Comment:
Btw headers may be of different type then simple object, it may be nested array or Headers instance, in which case this logic will be broken.
You should probably convert headers to a Headers instance as it will take care of handling any format type and then use it's method [`Headers.set()`](https://developer.mozilla.org/en-US/docs/Web/API/Headers/set) to set Content-Type header.